scalableminds/webknossos
# Segment multiselect

### Steps to test:
- Go to segments tab in any annotation, preferably one with a mesh file
- Press shift and select more than one segment
- Right-click to open the context menu like you are used to
- Select any action you would like to execute on all selected segments, such as loading all meshes
- Make sure this also works for meshes in different groups
- Make sure all actions are working as expected
- Make sure the visibility toggle for "Show /Hide meshes is working"
- Make sure selecting a group after having selected multiple segments opens a toast with a warning like in the skeleton tab
- After having selected meshes, open a group context menu and move all selected segments into the group

### TODOs:
- [x] self-review code before making the PR ready-for-review
- [x] fix case where group is selected after having selected multiple segments, but ctrl is not pressed (currently missing warning toast)

### Issues:
- partly fixes #6954 

------
(Please delete unneeded items, merge only when none are left open)
